packages/editor/src/components/copy-handler/index.js

```diff
@@ -75,13 +63,28 @@ export default compose( [
 		return {
 			hasMultiSelection: hasMultiSelection(),
 			selectedBlockClientIds,
+		};
+	} ),
+	withDispatch( ( dispatch, ownProps, select ) => {
+		const { getBlocksByClientId } = select( 'core/editor' );
+		const { removeBlocks } = dispatch( 'core/editor' );
+
+		return {
+			getSelectedBlocks: function() {
```

Comment:
Should we go further with that refactor and move both event handlers here?

Reply:
> Should we go further with that refactor and move both event handlers here?

I'm not totally following what's referred to by "both".